scheduler/bin/run-docker.sh

```diff
@@ -37,23 +37,34 @@ echo "Copying cook-executor from ${COOK_EXECUTOR_FILE} to ${SCHEDULER_EXECUTOR_D
 mkdir -p ${SCHEDULER_EXECUTOR_DIR}
 cp -f ${COOK_EXECUTOR_FILE} ${SCHEDULER_EXECUTOR_DIR}
 
+if [ -z "$(docker network ls -q -f name=cook_nw)" ];
+then
+    echo "Creating cook_nw network"
+    docker network create -d bridge --subnet 172.25.0.0/16 cook_nw
```

Comment:
Can we add a comment explaining why this is necessary?

Reply:
Done